src/MoviesContainer/MoviesContainer.test.js

```diff
@@ -51,4 +53,59 @@ describe('MovieContainer', () => {
     expect(mulanImage).toBeInTheDocument();
     expect(rogueImage).toBeInTheDocument();
   })
+  
```

Comment:
I'm glad we got more testing practice! I feel like I can write tests now. I'm not sure they are exactly correct or if they are always in the right place, but at least I am understanding them a bit now!

Reply:
I found it to be very rewarding!